tap_stripe/streams.py

```diff
@@ -368,6 +368,7 @@ class Plans(StripeStreamV2):
     replication_key = "updated"
     object = "plan"
     from_invoice_items = False
+    event_filter = "plan.*"
```

Comment:
for plans the reason that the `event_filter` is not set is because when a subscription price/plan is updated on the subscription itself the triggered event is not of type `plan.*` but `customer.subscription.updated` so we were missing those when using `plan.*`, I think it's not possible to request multiple event types, therefore we opted to fetch all events.